props/prop_base.h

```diff
@@ -741,16 +741,12 @@ class PropBase : CommandParser, Looper, protected SaberBase {
 
   virtual void CheckLowBattery() {
     if (battery_monitor.low()) {
-      // TODO: FIXME
       if (current_style() && !current_style()->Charging()) {
-	LowBatteryOff();
-	
-	if (millis() - last_beep_ > 5000) {
-#ifdef ENABLE_AUDIO
-	  // TODO: allow this to be replaced with WAV file
-	  talkie.Say(talkie_low_battery_15, 15);
-#endif
-	  STDOUT << "Battery low :" << battery_monitor.battery() << "\n";
+	if (SaberBase::IsOn()) {
```

Comment:
LowBatteryOff() already checks SaberBase::IsOn(), why do it again?

Reply:
Because we do something different if the saber is off.  So i still needed an on/off check.  If it is on, then I allow your function to run, which deals with turning the saber off.   If the saber is off, then I execute my logic to play the sound every 15 seconds